core/src/main/scala/org/apache/spark/shuffle/sort/SortShuffleManager.scala

```diff
@@ -48,7 +47,7 @@ private[spark] class SortShuffleManager(conf: SparkConf) extends ShuffleManager
       endPartition: Int,
       context: TaskContext): ShuffleReader[K, C] = {
     // We currently use the same block store shuffle fetcher as the hash-based shuffle.
-    new HashShuffleReader(
+    new MixedShuffleReader(
```

Comment:
Can we make it configurable (as the experimental feature for now) for turn on/off the `MixedShuffleReader`? Let's say we enable the `HashShuffleReader` by default, not the `MixedShuffleReader`